frontend/test/cypress/integration/product/add_to_cart.cy.ts

```diff
@@ -4,108 +4,65 @@ describe('product page add to cart', () => {
          '/products/spudger-retail-3-pack?variantid=39419992408154'
       );
       cy.findByTestId('cart-drawer-open').click();
-      cy.findAllByTestId('cart-drawer-item-count')
-         .should('exist')
-         .invoke('text')
-         .should('eq', '0');
+      cy.findAllByTestId('cart-drawer-item-count').should('have.text', 0);
       cy.findByTestId('cart-drawer-quantity').should('not.exist');
       cy.findByTestId('cart-drawer-close').click();
       cy.findByTestId('cart-drawer-close').should('not.exist');
    });
 
    it('Clicking Add To Cart Adds Items To Cart', () => {
-      const genArr = Array.from({ length: 5 }, (v, k) => k + 1);
-      cy.wrap(genArr).each((index) => {
+      for (let i = 1; i <= 5; i++) {
          cy.findByTestId('product-add-to-cart-button').click();
```

Comment:
theres a reason to use `cy.wrap()` here allegedly 

https://stackoverflow.com/questions/52212868/cypress-io-writing-a-for-loop
>This is the only reliable way, at least if the order of code execution is important. It's worth noting that - due to the asynchronous nature of cypress - a classical for loop will likely result in a non-deterministic execution order of the cy.* calls inside the loop.

Reply:
Ah I see, thanks for the link. I think this is working because the find...() function waits until each element is available, so when each iteration of the for loop is over, all assertions have been made. Maybe `.get()` doesn't wait

Reply:
in either case, i think its probably better form to use the `wrap()`

Reply:
I'm not sure there's a reason to block on this if the tests pass though? I might be misunderstanding.

Reply:
im not convinced that this change is necessary or desirable.

Reply:
i think why it still works is that each loop isn't dependent on the others. its like a map()

Reply:
Ok let me take one more shot at convincing you, and if you're not convinced I'll change it back.

> this change is necessary

This change is definitely not necessary since there's no functional difference. I think, however, that it's easier to tell what the bounds of the loop are. Another way that might be just as clear: `cy.wrap([1,2,3,4,5]).each()`. I think it's the `Array.from()` method that is kinda confusing (is it 0 indexed? does it include or exclude 5?).

> the asynchronous nature of cypress

Cypress (and javascript) are only async when we want them to be async. If we want actions to be done sequentially, we can tell cypress to do that. Here in the for-loop, all the actions are sequential. They will wait til the element is found in the DOM, or the test will fail because it couldn't find those elements, and it will abort execution. If someone later on introduces async code in the for loop, they will have to handle that appropriately.

---

So I guess I just find the `Array.from()` syntax hard to read, since it's so JS-specific. I'm open to other suggestions for iterating in a way that is easy to read.

Reply:
Ill say that i think using syntax that is language-specific isn't really a substantial reason to shy away from it because its easy to look up and well documented, but at this point im not going to continue to hold this up over 2 lines of inconsequential syntax

Reply:
I see your point. I think it's ok to disagree here, since functionally it's the same. I'm going to merge as is for expediency. But if you wanna change it back in the future, I'll give you CR on it.